zerver/openapi/zulip.yaml

```diff
@@ -7033,6 +7033,8 @@ paths:
                 - ""
             description: |
               You may pass the `client_gravatar` or `include_custom_profile_fields` query parameter as follows:
+      x-parameter-description: |
+        **Note**: The following parameters are all URL query parameters.
```

Comment:
Yeah, I think we should just delete these "URL query parameter" reminders.  If we want to have a notice of this form, we should compute whether to do it computationally.

Reply:
Can you start a thread in "#documentation" about these notices?  It's probably worth designing what we want to replace them with and opening an issue for our proposed solution.